src/graphics/webgpu/webgpu-bind-group-format.js

```diff
@@ -1,8 +1,16 @@
+import { Debug } from '../../core/debug.js';
+import { SAMPLETYPE_FLOAT, SAMPLETYPE_UNFILTERABLE_FLOAT, SAMPLETYPE_DEPTH } from '../constants.js';
+
 /** @typedef {import('../bind-group-format.js').BindGroupFormat} BindGroupFormat */
 /** @typedef {import('./webgpu-graphics-device.js').WebgpuGraphicsDevice} WebgpuGraphicsDevice */
 
 import { WebgpuUtils } from './webgpu-utils.js';
 
+const samplerTypes = { };
+samplerTypes[SAMPLETYPE_FLOAT] = 'filtering';
+samplerTypes[SAMPLETYPE_UNFILTERABLE_FLOAT] = 'non-filtering';
```

Comment:
Non filtering, is that a sampler object or is it a texture which can't perform fixed function bilinear/trilinear interpolation?